### ISSUE TITLE
Porn ads

### ISSUE THREAD

**Issue by bnainar** [Clefspeare13/pornhosts#357]
I believe this domain is an Adult(-related) domain --> that have to be blocked as..

## Domains
```
dq3yxnlzwhcys.cloudfront.net
r023m83skv5v.com
cdn.impactserving.com
impactserving.com
twinrdsrv.com
cdn.twinrdsrv.com
go.tmrjmp.com
glassmilheart.com
fierceharmlessjacket.com
favillafarine.com
venuegirtjive.com
js.wpnsrv.com
smart2cnjuh34jb.com
cdn.cloudvideosa.com
r023m83skv5v.com
cluster.awmserve.com
w4vecl1cks.com
```

Found on kaotic and efukt, screenshots are available in urlscan.io for respective domains

**Comment by spirillen** [comment]
If these are pure adware domain they should be added to a adware list such as https://github.com/mypdns/matrix/blob/master/source/adware/wildcard.list

**Comment by bnainar** [comment]
Serving ads directly and some redirects to other ads or malware domains

**Comment by spirillen** [comment]
> Serving ads directly and some redirects to other ads or malware domains

Then I'll recommend you to classify them, the best way is simply one issue = one domain, it is the best in both the long and short term

**Comment by bnainar** [comment]
go.tmrjmp.com is the redirecting one, others directly serve ads or call many scripts from other domains, blocking these domains is sufficient to block those other it calls too

**Comment by spirillen** [comment]
- `impactserving.com` already blocked by: https://www.mypdns.org/T2664 / https://github.com/easylist/easylist/pull/5523 / https://github.com/mypdns/matrix/commit/6a1de8182842dcef60f9118dc27274d19de842c4
  - `twinrdsrv.com` already blocked by https://www.mypdns.org/T2658 / https://www.mypdns.org/rMATRIX80fb99ba66a7e5059c7ae4c1430f178bc6e74526 (https://github.com/mypdns/matrix/commit/80fb99ba66a7e5059c7ae4c1430f178bc6e74526) / https://www.mypdns.org/rMATRIX9867fb5e7bc0755749135e6e013886b6dd231705 (https://github.com/mypdns/matrix/commit/9867fb5e7bc0755749135e6e013886b6dd231705)
  - `glassmilheart.com` already blocked by: https://www.mypdns.org/T4442 (https://github.com/mypdns/matrix/commit/7fef62377dd5aeaf092f8c92b40600a51c57e9c9#diff-38dbb19e022f62ddff6767f92d730f0ce43bf97a29d46619ed6cd5e624325581R15580) / EasyList (https://github.com/EasyList/EasyList/commit/6085ee6e95b#diff-1298f15fc987f32c8ee53ef08b4582d0861e6fecced9888537d7929b781c9ed2R3532)
  - `wpnsrv.com` Already blocked by: https://www.mypdns.org/T4151 / https://www.mypdns.org/rMATRIX3b7f3ab751ffd78bc470488760f69b70400efc81 (https://github.com/mypdns/matrix/commit/3b7f3ab751ffd78bc470488760f69b70400efc81)

This is by the way a duplicate of the following reports you have made several places
  - https://github.com/shreyasminocha/shady-hosts/issues/7
  - https://github.com/Sinfonietta/hostfiles/issues/137

Wontfix:
  - `cloudvideosa.com` will not be added as
    - The domain can't be confirmed
    - It have been removed from pgl.yoyo.org adware and tracking lists
